CSC631-MultiplayerGameDev-Group-Project/TeamFourProject
# Feat UI transition start to main

# About
This is an implementation from the Start Menu scene to the Main Menu scene.

# Testing this pull request
1. Go to Scenes/UI folder
2. Activate the StartMenuScene
3. Play the scene
4. Click the button 'Press to Play'
5. Notice the transition to the Main Menu scene with 3 buttons: Multiplayer, Settings, and Quit
